beacon_node/lighthouse_network/src/rpc/codec/ssz_snappy.rs

```diff
@@ -553,6 +564,14 @@ fn handle_v1_response<T: EthSpec>(
         Protocol::LightClientBootstrap => Ok(Some(RPCResponse::LightClientBootstrap(
             LightClientBootstrap::from_ssz_bytes(decoded_buffer)?,
         ))),
+        Protocol::LightClientOptimisticUpdate => {
+            Ok(Some(RPCResponse::LightClientOptimisticUpdate(Arc::new(
```

Comment:
I'm not sure if there's any good reason to try to decode this as we don't make outbound requests - it would be an unexpected scenario, and the response isn't going to be useful for us I think. I'll make this return an error and adda comment.

Reply:
We think it may be useful to handle outbound requests for testing, will cover this as part of #4940